src/command.js

```diff
@@ -181,10 +208,17 @@ GraphqurlCommand.flags = {
   // variables for the query
   variable: flags.string({
     char: 'v',
-    description: 'variables used in the query',
+    description: 'variables used in the query as key=value',
     multiple: true,
   }),
 
+  // variables for the query as JSON
+  variablesJSON: flags.string({
+    char: 'j',
+    description: 'variables used in the query as JSON key:value pairs',
```

Comment:
Description is not very clear. I suggest something along the lines of:

> query variables as a JSON string

Reply:
It'd be great if you could also change the description of `--variablesFile` to say that a JSON file is expected

Reply:
"query variables as a JSON string"  not clear 2 me, how about an example (not just a string either, nested json object)